scripts/IC.py

```diff
@@ -92,6 +94,32 @@ def client_install(comm, command, quit_ssh, device, _):
         command(f'interface gpon {client["frame"]} {client["slot"]}')
         command(f'ont delete {client["port"]} {client["onu_id"]}')
         log("Cliente no se agrego apropiadamente en OLT, eliminando...", "warning")
+
+        (client["device"], client["vendor"]) = type_finder(comm, command, client)
+        if client["vendor"] == "BDCM":
+            client['device'] = client['vendor']
+        # log(f"El tipo de ONT del cliente es {client['device']}", "ok")
+        
+        client['status'] = "offline"
+        client['state'] = "deactivate"
+
+        for key in client_payload:
+            client_payload[key] = client[key]
+
+        client_payload["olt"] = device
+        client_payload["fsp"] = f'{client["frame"]}/{client["slot"]}/{client["port"]}'
+        client_payload[
+            "fspi"
+        ] = f'{client["frame"]}/{client["slot"]}/{client["port"]}/{client["onu_id"]}'
+        client_payload["spid"] = 0
+        client_payload["device"] = client["device"]
+        payload_add["data"] = client_payload.copy()
+        req = db_request(endpoints["add_client"], payload_add)
```

Comment:
Did you want to delete or add the client?

Reply:
adding the registry to the DB of the user that was not installed but was added in the olt anyway so it could be managed later in the IT branch